### PR TITLE
Fix zapier

### PR DESCRIPTION
**Pull request by martmull**
This PR adapts zapier request to match the new production app requirements
- fix tests
- fix requests
- minor improvements

### CODE REVIEW

**Review by charlesBochet — COMMENTED** [review]
Left a few comments!

File: packages/twenty-zapier/src/creates/create_company.ts [hunk 1]
```diff
@@ -47,29 +47,53 @@ export default {
         altersDynamicFields: false,
       },
       {
-        key: 'linkedinUrl',
-        label: 'Linkedin',
+        key: 'linkedinLink__url',
```
[the remark this answers is not in the document]
Comment (charlesBochet) [reply packages/twenty-zapier/src/creates/create_company.ts:50]: I guess I have my answer below :p

[the remark this answers is not in the document]
Comment (martmull) [reply packages/twenty-zapier/src/creates/create_company.ts:50]: of course, i tried to do that, I spent some time to figure out to make it work but did not succeed. Finally end with this flattened format

**Review by charlesBochet — APPROVED** [review]
LGTM!

**Pull request merged**